### ISSUE TITLE
[FE] Profile:  508-defect-2 [Screen Readers, Cognition]: Focus is lost on error messages

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#28953]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

Issue Title: 508 Defect 2, Screen Readers, Cognition, focus gets lost on error messages.
Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

## User Story or Problem Statement:
As a screen reader user, I expect to be able to easily dispatch error messages.

## Details
When editing contact information, in the event that the information can't be updated, an error message will be triggered. Currently, focus is not managed so the user has to navigate through several buttons to find "Close notification." Once the notification is closed, the user cannot proceed until the "update" function is canceled, but once again focus is placed at the top of the page resulting in more tedious navigation.

## Acceptance Criteria
Focus is properly managed throughout the process of dismissing error message.

## Proposed Solution (if known)
Focus should be placed on the "Close notification" button. Once that button is selected, either the "Update" function should be canceled or focus should be placed on the "Cancel" button.

**Comment by tmitchellgcio** [comment]
@ajakabcin ready for validation

**Comment by AngelaFowler82** [comment]
When the error message popps up, focus is placed on "Close." When I close the message, I am taken back to the update screen and focus is placed on the "Update" button. The only thing I would suggest would be to place focus on the "phone number" edit field upon closing the error. Other than that, it is working much better.

**Comment by ajakabcin** [comment]
Thank you @AngelaFowler82! @tmitchellgcio see feedback above : )

**Comment by tmitchellgcio** [comment]
The newly merged pr makes this change @AngelaFowler82 can you confirm?

**Comment by AngelaFowler82** [comment]
Confirming that the change took effect. I'll go ahead and close this now.
